README.md

```diff
@@ -1,2 +1,30 @@
 # Bitcoin Design - Meta
-`community`, `project-management`, `process`
\ No newline at end of file
+`community`, `project-management`, `process`
+
+Anything related to group consensus and coordination of the Bitcoin Design Community.
+
+### Getting Started
+1. Pick existing [project](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit) in the space and do something small
+2. If you are passionate. Just do it. Worst case you publish it on your own
+3. Contribute to parts of the design guide and website even if it is a slower process to get things approved
+
```

Comment:
IMO we should remove negative language like "even if it is a slower process" ... let's be optimistic when introducing new people to the project :)